decoders/boliou_point_decoder.py

```diff
@@ -0,0 +1,134 @@
+'''
+    Miaoye Que, Last Updated Jan 21, 2019
+    Decodes point data in Boliou
+'''
+
+import re
+
+from decoders.point_decoder import PointDecoder
+
+
+class BoliouPointDecoder(PointDecoder):
+
+    @staticmethod
+    def decode_building_name(attr_dict):
+        return "Boliou"
+
+    @staticmethod
+    def decode_device_name(attr_dict):
+        return "unknown"
+
+    @staticmethod
+    def decode_device_desc(attr_dict):
+        return "unknown"
+
+    @staticmethod
+    def decode_room_name(attr_dict):
+        sub_names = attr_dict["Point Name"][0].split(':')
+        location_name = sub_names[0]
+        location_units = location_name.split('.')
+        if len(location_units) == 4:
+            room_name = location_units[3]
+            if "RM" in room_name:
+                room_name_list = re.findall(r'\d+', room_name)
+                return room_name_list[0] if room_name_list else 'unknown'
+        return "unknown"
+
+    @staticmethod
+    def decode_room_floor(attr_dict):
```

Comment:
We currently store the floor as an int the database. Chris, Ethan, and I aren't exactly sure how to resolve the problem of ground floors and basements and inconsistencies between buildings, but I do this this needs to be an int or we need to store floors differently.